components/SnippetForm.js

```diff
@@ -127,7 +136,7 @@ function SnippetForm({ onSubmit, formName, defaultData, defaultTags }) {
                   as="input"
                   type="text"
                   id={linkObject.id}
-                  placeholder="www."
+                  placeholder="Enter URL"
```

Comment:
placeholders usually bring some a11y issues:
- low contrast
- mistakenly considered to be filled out inputs
- disappears on input

in this case, if you want to keep it, I would prefill with a relevant value like: `https://www.google.com` to show the real format and not give any descriptive content.